src/main/java/org/eolang/speco/Speco.java

```diff
@@ -90,8 +90,7 @@ public void exec() throws IOException {
             source = this.input;
         }
         for (final Path path : Files.newDirectoryStream(source)) {
-            final XML before;
-            before = Speco.getParsedXml(new XMLDocument(Files.readString(path)));
+            final XML before = Speco.getParsedXml(new XMLDocument(Files.readString(path)));
             final String after;
             if (this.eolang) {
                 after = new XMIR(
```

Comment:
@advasileva You have code duplication here `Speco.applyTrain(before).toString()`, apparently you can move that upper